DQM/SiStripMonitorClient/plugins/SiStripCertificationInfo.cc

```diff
@@ -1,30 +1,79 @@
+#include "CalibFormats/SiStripObjects/interface/SiStripDetCabling.h"
+#include "CalibTracker/Records/interface/SiStripDetCablingRcd.h"
+#include "CondFormats/DataRecord/interface/RunSummaryRcd.h"
+#include "CondFormats/RunInfo/interface/RunInfo.h"
+#include "DataFormats/Histograms/interface/DQMToken.h"
+#include "DataFormats/TrackerCommon/interface/TrackerTopology.h"
+#include "DQM/SiStripCommon/interface/SiStripFolderOrganizer.h"
+#include "DQM/SiStripMonitorClient/interface/SiStripUtility.h"
+#include "DQMServices/Core/interface/DQMStore.h"
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/LuminosityBlock.h"
+#include "FWCore/Framework/interface/Run.h"
+#include "FWCore/Framework/interface/EventSetup.h"
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
-#include "FWCore/ServiceRegistry/interface/Service.h"
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ServiceRegistry/interface/Service.h"
+#include "Geometry/Records/interface/TrackerTopologyRcd.h"
 
-#include "DQMServices/Core/interface/DQMStore.h"
-#include "DQM/SiStripCommon/interface/SiStripFolderOrganizer.h"
-#include "DQM/SiStripMonitorClient/interface/SiStripUtility.h"
-#include "SiStripCertificationInfo.h"
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <vector>
+#include <map>
 
-#include "DataFormats/Histograms/interface/DQMToken.h"
+class SiStripDetCabling;
 
-//Run Info
-#include "CondFormats/RunInfo/interface/RunInfo.h"
+class SiStripCertificationInfo : public edm::one::EDAnalyzer<> {
```

Comment:
Is this the right fix?
I think this should be moved to `DQMEDHarvester` instead.